code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -231,7 +225,7 @@ Auto Patrol: []"},
 	add_logs(src,C,"stunned")
 	if(declare_arrests)
 		var/area/location = get_area(src)
-		speak("[arrest_type ? "Detaining" : "Arresting"] level [threat] scumbag <b>[C]</b> in [location].", radio_channel)
+		speak("Arresting level [threat] scumbag <b>[C]</b> in [location].", radio_channel)

```

Comment:
Please keep the detain function, remove it from the interface & topic calls, allow mappers/admins to use it if they wish.